app/src/main/kotlin/com/wire/android/ui/home/newconversation/common/CreateGroupErrorDialog.kt

```diff
@@ -74,7 +74,7 @@ fun CreateGroupErrorDialog(
                 }
                 addStringAnnotation(
                     tag = MarkdownConstants.TAG_URL,
-                    annotation = stringResource(id = R.string.url_support),
+                    annotation = stringResource(id = R.string.url_message_details_offline_backends_learn_more),
```

Comment:
are those link related to a backend or the same for all ?

Reply:
Is one general `url` for federation support. the issue was we were using the general wire support link instead.